tests/phpunit/tests/image/editorImagick.php

```diff
@@ -935,4 +935,67 @@ public static function data_alpha_transparency_is_preserved_after_resize() {
 			),
 		);
 	}
+
+	/**
+	 * Test AVIF quality filters.
+	 *
+	 * @ticket 61614
+	 *
+	 * Temporarily disabled until we can figure out why it fails on the Trixie based PHP container.
+	 * See https://core.trac.wordpress.org/ticket/63932.
+	 * @requires PHP < 8.3
+	 */
+	public function test_quality_with_avif_conversion_file_sizes() {
+		$temp_dir = get_temp_dir();
+		$file     = $temp_dir . '/33772.jpg';
+		copy( DIR_TESTDATA . '/images/33772.jpg', $file );
+
+		$editor = wp_get_image_editor( $file );
```

Comment:
since this is called on the jpeg it will select the default image editor, usually Imagick unless a filter is being applied.